docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -276,19 +282,20 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv4OrDefault(string)
+## toIPv4OrDefault
 
-Same as `toIPv4`, but if the IPv4 address has an invalid format, it returns `0.0.0.0` (0 IPv4).
+Same as `toIPv4`, but if the IPv4 address has an invalid format, it returns `0.0.0.0` (0 IPv4), or the provided IPv4 default.
 
 **Syntax**
 
 ```sql
-toIPv4OrDefault(value)
+toIPv4OrDefault(value [, default])
```

Comment:
Can we give the first argument in l. 254 and in this line the same name? ("value" vs. "string")